front-end/src/components/navigation/navigation-icon/NavigationIcon.js

```diff
@@ -13,7 +13,7 @@ class NavigationIcon extends Component {
                 <SvgIcon isActive={this.props.match && this.props.match.isExact ? true : false} name={this.props.iconName} />
```

Comment:
This component needs not knowing about the existence of react-router
You can either receive isActive from `Navigation` directly, or use this magic:
https://reacttraining.com/react-router/web/api/NavLink

(React Router's NavLink component knows if it is active or not automatically. You dont need to do the complex path checking yourself)

Reply:
Or pass isActive from NavLink